resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/resources/IFile.java

```diff
@@ -1163,6 +1173,93 @@ public interface IFile extends IResource, IEncodedStorage, IAdaptable {
 	 */
 	void setContents(IFileState source, int updateFlags, IProgressMonitor monitor) throws CoreException;
 
+	/**
+	 * Preferable equivalent of calling
```

Comment:
Can elaborate on this why it is preferable?

Reply:
done.